src/main/java/org/opentripplanner/graph_builder/module/NearbyStopFinder.java

```diff
@@ -56,16 +57,8 @@
      * network or straight line distance based on the presence of OSM street data in the graph.
      */
     public NearbyStopFinder(Graph graph, double radiusMeters) {
-        this (graph, radiusMeters, graph.hasStreets);
-    }
-
-    /**
-     * Construct a NearbyStopFinder for the given graph and search radius.
-     * @param useStreets if true, search via the street network instead of using straight-line distance.
-     */
-    public NearbyStopFinder(Graph graph, double radiusMeters, boolean useStreets) {
```

Comment:
This method signature was only ever called from the above constructor, so I combined these constructors into one.